translations/en_US.json

```diff
@@ -83,7 +83,7 @@
   "asset_creators_title": "Creators",
   "profile_balance_of": "of {balance} {symbol}",
   "modal_switch_network_cancel": "Cancel",
-  "buy_lyx_card_transak_description": "Available across 170 cryptocurrencies on 75+ blockchains via cards, bank transfers and other payment methods in 160 countries.",
+  "buy_lyx_card_transak_description": "Transak provides leading Web3 on-ramp services, enabling users from over 180 countries to access $LYX through various fiat payment methods.",
```

Comment:
Is proper to use `$LYX`?

Reply:
Jonathan and Biyan provided this new description, so I guess we can use `$LYX` ^^'